libraries.json

```diff
@@ -130,7 +130,7 @@
     "name_pretty": "Certificate Authority",
     "product_documentation": "https://cloud.google.com/certificate-authority-service",
     "client_documentation": "https://googleapis.dev/nodejs/security-private-ca/latest/index.html",
-    "issue_tracker": "",
+    "issue_tracker": "https://b.corp.google.com/components/402191#basic",
```

Comment:
@sofisl this is an internal only URL :) We probably need to go back to the certificate authority library, and update this with a public url.  I usually start looking here:
https://cloud.google.com/support/docs/issue-trackers

If I can't find something, I use the github issues url